docs/using-radicle/view-share.md

```diff
@@ -3,81 +3,85 @@ id: view-share
 title: View and share projects
 ---
 
-After you've pushed your project to the Radicle network, it's available for viewing on the Radicle [web
-interface](https://app.radicle.network), which helps you discover new projects hosted on the network. The web interface
-is organized by either the seed node a project is synced with or the organization it's published under, such as
-Radicle's [alt-clients](https://app.radicle.network/alt-clients.radicle.eth) organization.
+After you've pushed your project to the Radicle network, you can view it using either the Upstream desktop app or the
+web interface. They have a similar user interface and experience &mdash; the primary difference is that Upstream
```

Comment:
The web interface is about to support patches, wouldn't limit patches view to Upstream
https://github.com/radicle-dev/radicle-interface/pull/232

Reply:
We might want to update the docs once this is live to give users a screenshot on what the patches view would look like in the web client